<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonDT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Double Transposition</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonA52">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>A5/2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCFB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ciphertext Feedback</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelSelectAlgorithm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Algorithm:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFSW">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn FSW on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonManual">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file manually</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSend">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRecieve">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Recieve file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
